app/forms/conversion/create_project_form.rb

```diff
@@ -36,11 +38,17 @@ class NegativeValueError < StandardError; end
   validate :establishment_exists, if: -> { urn.present? }
   validate :trust_exists, if: -> { incoming_trust_ukprn.present? }
 
+  validates :directive_academy_order, inclusion: {in: ["true", "false"]}
```

Comment:
These are strings not booleans, as they're coming from a form I think? Bools didn't work here. Happy to be corrected if I've done it wrong!